src/invert/laplace/impls/mumps/mumps_laplace.cxx

```diff
@@ -34,10 +34,11 @@
 #include <msg_stack.hxx>
 #include <cmath>
 
-LaplaceMumps::LaplaceMumps(Options *opt) : 
+LaplaceMumps::LaplaceMumps(Options *opt, Mesh *passmesh) : 
```

Comment:
`passmesh` can just be `localmesh`, there's no name conflict when used in a initialiser list like here.